packages/flutter_test/lib/src/goldens.dart

```diff
@@ -174,7 +174,7 @@ abstract class WebGoldenComparator {
   /// is left up to the implementation class. For instance, some implementations
   /// may load files from the local file system, whereas others may load files
   /// over the network or from a remote repository.
-  Future<bool> compare(Element element, Size size, Uri golden);
+  Future<bool> compare(dynamic element, dynamic size, Uri golden);
```

Comment:
Though its useful for debugging, I don't think this is a good long-term change since we lose type safety on the APIs. I would recommend exploring how you can expose the same logic without changing the existing APIs

Reply:
maybe instead of having conditional import in this file, it's really up to the caller to either import web or io. That will remove this problem too. WDYT?

Reply:
done. PTAL